compiler/passes/ResolveScope.cpp

```diff
@@ -650,9 +650,15 @@ Symbol* ResolveScope::lookupForImport(Expr* expr, bool isUse) const {
         if (isUse)
           USR_FATAL(expr, "use must name a module or enum ('%s' is neither)",
                           sym->name);
-        else
+        else if (relativeScope == NULL || relativeScope == this)
           USR_FATAL(expr, "import must name a module ('%s' is not a module)",
-                          sym->name);
+                    sym->name);
+        else {
+          // It's okay for the symbol following a super to not be a module, so
+          // long as it is the last symbol in the import statement...
+          retval = sym;
+          break;
```

Comment:
Is there a test of `import super.Mod.super.SubMod.x` ? Should we see an error there, or is it supposed to work? It's not clear to me that this code handles that case correctly so I'd appreciate it if you can add a test / find one that exists.

Reply:
It looks like there isn't already a test of this so far as I can see.  In terms of what it should do, I would be okay with it working for the following hierarchy:
```chapel
module M {
  module Mod { ... }
  module SubMod {
     var x: int;
  }
  module User {
    import super.Mod.super.SubMod.x;
    proc main() {
      writeln(x);
    }
  }
}
```
but would also be okay with it not working, since that's not really a clear path for people reading the code.  As written, I think it's pretty similar to `../Mod/../SubMod` in file paths and that's something that's allowed generally.

What actually happens is that we get `error: Cannot find symbol 'super' in module 'Mod'`, it looks like due to looking for the name "super" in the module.  This is not changed by my branch, it behaves the same on master - if we want it to work, I think that should be a separate PR